cibyl/sources/elasticsearch/api.py

```diff
@@ -38,29 +38,36 @@
 class ElasticSearch(ServerSource):
     """Elasticsearch Source"""
 
-    def __init__(self, driver: str = 'elasticsearch',
+    def __init__(self: object, driver: str = 'elasticsearch',
```

Comment:
There are still a few self annotations in the elasticsearch modules

Reply:
`(venv3.10) [2022-06-27 08:54:17] {afuscoar@afuscoar} (~/.Redhat/Projects/GitHub/cibyl/cibyl) (feature/add-teardown-methods)$ -> for file in $(grep -Ril 'self: object'); do sed -i 's/self: object/self/' $file; done` :D

Done.